proposed/2023/allow-version-property-in-projectreference.md

```diff
@@ -0,0 +1,107 @@
+# Allow users to define version ranges for ProjectReferences
+
+- [Martin Ruiz](https://github.com/martinrrm)
+- Start Date (2023-01-01)
+- [5556](https://github.com/NuGet/Home/issues/5556)
+
+# Summary
+
+Add a `Version` to `ProjectReference` tag in CSPROJ, to allow customers to specify the referenced project version in the `.nupkg` and `nuspec` files when doing a pack command.
+
+# Motivation
+
+When using `ProjectReference` there is no option to define a Version to the reference like in `PackageReference` and the version will always be defined as `>= ProjectVersion`, this results in customers manually modifying the nuspec file if they want to declare a different version than the project version.
```

Comment:
Could you be able to add little bit more details on exactly what is the project version here?
What happens if I change different branch which have same version but with different content? 
For packages we have dedicated global packages and for same `id+version` content doesn't mutate. Here it mutates, so what happens now for same project version with different content?

Reply:
For project version I mean the value of `<Version></Version>` in the CSPROJ. 

About `what happens now for same project version with different content?` I'll do more testing but I;m not sure if I understand your concern.

For example if ProjectA has a `ProjectReference` to `MyReferencedProject` like this: `<ProjectReference Include="MyReferencedProject" Version="[9.0.0, 13.0.2)"  /> ` and you change the content in MyReferencedProject but not the version, you mean the package content should not change?